build.sh

```diff
@@ -369,7 +374,7 @@ fi
 # build cudf_kafka Python package
 if hasArg cudf_kafka; then
     cd ${REPODIR}/python/cudf_kafka
-    SKBUILD_CONFIGURE_OPTIONS="-DCMAKE_LIBRARY_PATH=${LIBCUDF_BUILD_DIR}" \
+    SKBUILD_CONFIGURE_OPTIONS="-DCMAKE_PREFIX_PATH=${INSTALL_PREFIX} -DCMAKE_LIBRARY_PATH=${LIBCUDF_BUILD_DIR} -DCMAKE_CUDA_ARCHITECTURES=${CUDF_CMAKE_CUDA_ARCHITECTURES} ${EXTRA_CMAKE_ARGS}" \
```

Comment:
Ah. First, see the comment above. https://github.com/rapidsai/cudf/pull/14292#discussion_r1390084465

The issue is that if libcudf is _not_ found, the `cudf_kafka` wheel has to compile libcudf, too. We probably _never_ want to ship cudf_kafka as a wheel for this reason -- only as a conda package where we can know that libcudf will be available. I'm torn about how to handle this case since we need scikit-build/CMake packaging (which implies wheel compatibility if we're keeping it aligned with other packages) but we would never want to ship all the bits for libcudf.so in both the cudf wheel and in a hypothetical cudf_kafka wheel.

Reply:
@vyasr It would probably be easiest for us to review this together on a call, so that we can get it shipped as early as possible in burndown and talk over the confusing pieces.

Reply:
For the record, we discussed offline and concluded that the easiest path forward here is to have cudf_kafka be much simpler than our other RAPIDS packages by not supporting building the C++ components from the Python build. This won't work for our typical wheel workflows, but we have no plans for building wheels of this package right now so that's a non-issue, and it will dramatically simplify these build scripts right now.